examples/core/transform-feedback-instanced/app.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
Nit: Just inline this CSS in the HTML to avoid so many files and keep example compact?

```<style> ... </sgyle>```

Reply:
Looks like this file ins't needed, style tag already exists in html, removed css file.

Reply:
QQ: Was this copied from other examples internally? Could the same optimization be made in other places?